src/graph/topo.cc

```diff
@@ -371,7 +371,12 @@ ncclResult_t ncclTopoAddNic(struct ncclXmlNode* xmlNic, struct ncclTopoSystem* s
 }
 
 ncclResult_t ncclTopoAddGpu(struct ncclXmlNode* xmlGpu, struct ncclTopoSystem* system, struct ncclTopoNode* gpu) {
+#if defined(__HIP_PLATFORM_HCC__) || defined(__HCC__) || defined(__HIPCC__)
```

Comment:
I think __HIP_PLATFORM_HCC__ is deprecated, we are supposed to use __HIP_PLATFORM_AMD__

Reply:
yes, this is just to be consistent with other part of code. We need another patch to change them all.

Reply:
Maybe next patch is good candidate?

Reply:
I thought we had this already fixed last fall, because I remember that somebody had a pr for that. But it looks like it didn't get merged, U